client/verta/tests/unit_tests/test_runtime.py

```diff
@@ -89,15 +75,16 @@ def test_logs_are_clean_on_entry():
         assert ctx.logs() == {}
 
 
-def test_logs_are_clean_on_exit():
-    """ Thread local var for logs is a blank dict after exiting context. """
-    with runtime.context() as ctx:
+def test_logs_attribute_removed_on_exit():
+    """ Thread local var for logs is removed after exiting the context. """
+    with runtime.context():
         runtime.log('fake_log', 'fake_value')
-    assert runtime._get_thread_logs() == {}  # outside the context
+    assert not hasattr(runtime._THREAD, 'logs')  # outside the context
```

Comment:
Optionally, we might want to add a

```python
ctx.logs()
```

before the `hasattr` check, since that was the exact issue we were seeing. But probably not necessary, now that we've removed the behavior altogether.

Reply:
Good call.  Verifying that logs are there before we verify that they get removed makes the test a bit more complete.  >> Done

Reply:
I actually meant a `ctx.logs()` _outside_ the context, since the original bug was that calling `ctx.logs()` after `context.__exit__()`! But it's still probably fine either way 😆